pkg/controllers/resourcechange/resourcechange_controller.go

```diff
@@ -115,68 +127,94 @@ func (r *Reconciler) handleUpdatedResourceForClusterResourcePlacement(key keys.C
 		klog.ErrorS(err, "Failed to trigger affected placements for updated cluster resource", "obj", key)
-		return ctrl.Result{}, err
+		return err
 	}
-	return res, nil
+	return nil
 }
 
 // handleUpdatedResourceForResourcePlacement handles the updated resource for resource placement.
-func (r *Reconciler) handleUpdatedResourceForResourcePlacement(key keys.ClusterWideKey, clusterObj runtime.Object, isClusterScoped bool) (ctrl.Result, error) {
+func (r *Reconciler) handleUpdatedResourceForResourcePlacement(key keys.ClusterWideKey, clusterObj runtime.Object, isClusterScoped bool) error {
 	if isClusterScoped {
-		return ctrl.Result{}, nil
+		return nil
 	}
 
 	klog.V(2).InfoS("Find resourcePlacement that selects the namespace scoped object", "obj", key)
-	res, err := r.triggerAffectedPlacementsForUpdatedClusterRes(key, clusterObj.(*unstructured.Unstructured), false)
-	if err != nil {
+	if err := r.triggerAffectedPlacementsForUpdatedRes(key, clusterObj.(*unstructured.Unstructured), false); err != nil {
 		klog.ErrorS(err, "Failed to trigger affected placements for updated resource", "obj", key)
-		return ctrl.Result{}, err
+		return err
 	}
-	return res, nil
+	return nil
 }
 
 // handleUpdatedResource handles the updated resource and triggers the affected placements.
 func (r *Reconciler) handleUpdatedResource(key keys.ClusterWideKey, clusterObj runtime.Object, isClusterScoped bool) (ctrl.Result, error) {
-	if _, err := r.handleUpdatedResourceForClusterResourcePlacement(key, clusterObj, isClusterScoped); err != nil {
+	if err := r.handleUpdatedResourceForClusterResourcePlacement(key, clusterObj, isClusterScoped); err != nil {
 		klog.ErrorS(err, "Failed to handle updated resource for placement", "obj", key)
 		return ctrl.Result{}, err
 	}
-	if _, err := r.handleUpdatedResourceForResourcePlacement(key, clusterObj, isClusterScoped); err != nil {
+	if err := r.handleUpdatedResourceForResourcePlacement(key, clusterObj, isClusterScoped); err != nil {
 		klog.ErrorS(err, "Failed to handle updated resource for resource placement", "obj", key)
 		return ctrl.Result{}, err
 	}
 	klog.V(2).InfoS("Successfully handled updated resource", "obj", key)
 	return ctrl.Result{}, nil
 }
 
-// triggerAffectedPlacementsForDeletedClusterRes find the affected placements for a given deleted cluster scoped resources
-func (r *Reconciler) triggerAffectedPlacementsForDeletedClusterRes(res keys.ClusterWideKey) (ctrl.Result, error) {
-	if r.PlacementControllerV1Alpha1 != nil {
-		crpList, err := r.InformerManager.Lister(utils.ClusterResourcePlacementV1Alpha1GVR).List(labels.Everything())
-		if err != nil {
-			klog.ErrorS(err, "failed to list all the v1alpha1 cluster placement", "obj", res)
-			return ctrl.Result{}, err
+// triggerAffectedPlacementsForDeletedRes find the affected placements for a given deleted resources.
+func (r *Reconciler) triggerAffectedPlacementsForDeletedRes(key keys.ClusterWideKey, isClusterScope bool) (ctrl.Result, error) {
```

Comment:
Rename to `triggerCRP`? To be consistent with `triggerAffectedPlacementsForUpdateRes`.

Reply:
triggerAffectedPlacementsForDeletedRes is consistent with triggerAffectedPlacementsForUpdatedRes?